iris_grib/message.py

```diff
@@ -33,9 +33,24 @@
 from iris._lazy_data import as_lazy_data
 from iris.exceptions import TranslationError
 
+
 _SUPPORTED_GRID_DEFINITIONS = (0, 1, 5, 10, 12, 20, 30, 40, 90)
 
 
+def grib_get_array_wrapper(f):
```

Comment:
@lbdreyer I guess we should really make this a private function i.e. `_grib_get_array_wrapper`.